elements/lisk-p2p/src/p2p.ts

```diff
@@ -136,6 +138,8 @@ export const DEFAULT_PEER_PROTECTION_FOR_USEFULNESS = 0.068;
 export const DEFAULT_PEER_PROTECTION_FOR_LONGEVITY = 0.5;
 export const DEFAULT_MIN_PEER_DISCOVERY_THRESHOLD = 100;
 export const DEFAULT_MAX_PEER_DISCOVERY_RESPONSE_SIZE = 1000;
+const SECRET_LENGTH = 4;
+export const DEFAULT_SECRET = randomBytes(SECRET_LENGTH).readUInt32BE(0);
```

Comment:
Please, explain what this secret is for and don't export it because we don't use it anywhere else.

Reply:
Related to PR #3976 https://github.com/LiskHQ/lisk-sdk/pull/4030#discussion_r309754614

Reply:
@diego This is already added in [netgrouping PR](https://github.com/LiskHQ/lisk-sdk/pull/4023). It is used as part of the hash for bucketing and netgrouping.